doc/source/dev_guide/testing.rst

```diff
@@ -0,0 +1,6 @@
+Testing Satpy-based applications
+================================
+
+The ``satpy.testing`` modules provides tools for writing tests of applications that use Satpy.
+
+- To allow Scene creation and loading fake datasets without reading any data file: :func:`~satpy.testing.fake_satpy_reading`
```

Comment:
```suggestion
- To allow Scene creation and loading fake datasets without reading any data files: :func:`~satpy.testing.fake_satpy_reading`
```

I think English-wise it sounds better to say "any data files" ("a data file" could work, but I think is less accurate here).

Reply:
Fixed